Recruit-Training/Advanced-Recruit-Training/Telemetry-Teaser/src/App.tsx

```diff
@@ -1,25 +1,118 @@
+import { useState } from "react";
+
 import BatteryInput from "~/components/batteryInput";
 import Header from "~/components/header";
 import SpeedInput from "~/components/speedInput";
 import WeatherInput from "~/components/weatherInput";
 
 const App = () => {
-  const calculateRange = () => {
-    return;
-  }
+  const [speed, setSpeed] = useState<number | undefined>(0);
```

Comment:
speed, battery, and weather are all together. can we create a single state that represents all 3 of the inputs?

example const [inputs, setInputs] = useState({}), where the object is the key value pair for each of the 3 inputs?